app/client/components/helpCentre/HelpTopicBox.tsx

```diff
@@ -67,7 +67,7 @@ const linkAnchorStyle = css`
   }
 `;
 
-const linkArrowStyle = css`
+export const linkArrowStyle = css`
```

Comment:
If we're going to create a new `helpCentreStyles` file, then this would also be a good candidate for adding in to it :)